assets/css/ie-editor.css

```
@@ -1364,26 +1364,38 @@ p.has-background {
 	padding: 40px 0;
 	margin-left: 0;
 	margin-right: 0;
-	text-align: left;
-	border-top-color: transparent;
-	border-top-width: 0;
-	border-bottom-color: transparent;
-	border-bottom-width: 0;
+	text-align: center;
+	border-top-color: #28303d;
+	border-top-style: solid;
+	border-top-width: 3px;
+	border-bottom-color: #28303d;
+	border-bottom-style: solid;
+	border-bottom-width: 3px;
 	color: #28303d;
 }
 
+.wp-block-pullquote::before {
+	color: currentColor;
+	content: "“";
```

Generally unicode chars are used for CSS `content` if any non-standard chars are added there.
```suggestion
	content: "\201C";
```

---

Thank you! Changed that one.